roles/ceph-osd/tasks/system_tuning.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: disable osd directory parsing by updatedb
+  command: updatedb -e /var/lib/ceph
+  changed_when: false
+  failed_when: false
+
+- name: disable transparent hugepage until next reboot
+  command: hugeadm --thp-never
```

Comment:
We need a task to ensure that the package that provides hugeadm is installed, unless it's like a core dependency of everything.